RNTester/RNTesterPods.xcodeproj/xcshareddata/xcschemes/RNTester.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really need those xcscheme files to be checked in?

Reply:
If I don’t check them in, there is no schema to run. This seems to be the case on master too I think, so I suspect something got messed up somewhere along the way *before* these changes.

Reply:
Hmm, I don't think this file should be checked in. We should update the .gitignore under RNTester to ignore .xccheme -- I think this scheme gets generated when you pod install?